micro-deps/build.gradle

```diff
@@ -22,6 +22,31 @@ project(':micro-deps-root:micro-deps') {
     }
 }
 
+project(':micro-deps-root:micro-deps-jax-rs') {
```

Comment:
There are 2 approaches to deal with that:
1. Module with interface (*jax-rs) and 1 module per implementation: *-jax-rs-jersey, *-jax-rs-hystrix etc,
2. A single *-jax-rs module with jersey, hystix etc declared as optional dependencies.
I was not sure what to choose until lately, however now it is clear that option 2 will not work.
Therefore, I will rename the module to 'micro-deps-jax-rs-jersey' for now